pkg/api/helper/helpers.go

```diff
@@ -107,10 +107,13 @@ func IsResourceQuotaScopeValidForResource(scope api.ResourceQuotaScope, resource
 var standardContainerResources = sets.NewString(
```

Comment:
FYI @derekwaynecarr if this patch is accepted then the new [hugepage resources](https://github.com/kubernetes/community/pull/837) need to be added to this whitelist.